include/ck/utility/reduction_operator.hpp

```diff
@@ -112,12 +121,21 @@ struct Mul
     __host__ __device__ inline constexpr void operator()(T& a, T b) const
     {
         static_assert(is_same<T, float>::value || is_same<T, double>::value ||
-                          is_same<T, int32_t>::value || is_same<T, half_t>::value,
+                          is_same<T, int32_t>::value || is_same<T, half_t>::value ||
+                          is_same<T, f8_t>::value || is_same<T, int8_t>::value,
```

Comment:
You don't have to check for `f8_t` here since you provide specialization for this type. 
```suggestion
                          is_same<T, int8_t>::value,
```